docs/docs/concepts/deployment.md

```diff
@@ -0,0 +1,84 @@
+
+For more complicated agent deployments (whether using LangGraph or not), you will run into some issues. The rest of this guide will discuss those issues.
+
+> 📘 LangGraph Platform
+>
+> This guide covers concepts related to deployment in general. [LangGraph Platform](./langgraph_platform.md) is an opinionated way to deploy LangGraph agents built by the LangChain team, and handles all the issues raised here.
+
+## Streaming support
+
+[Streaming](streaming.md) is a key part of LLM applications. It is usually necessary to stream back both tokens and intermediate steps of longer running agents. This is because LLM applications can take a while to run end-to-end, and so streaming helps show the user that something is happening.
+
+LangGraph support [multiple different streaming modes](streaming.md). When deploying, you will want to make sure that you expose the relevant endpoints.
+
+## Background runs
+
+As your LLM application gets more complicated and starts taking longer to run, you may no longer want to stream back results. This is because these runs may take a while to finish and it may not make sense to hold open a connection for all that time.
+
+As a result, you may also want to consider exposing an endpoint to run your LangGraph as a background run. When doing this, you will need to expose an endpoint that you can poll to check if your run is finished, or what its status is.
+
+## Long runs
+
+In addition to adding endpoints for [background runs](#background-runs), you also need to make sure your infrastructure can support long lasting runs. Some web infrastructure is optimized for syncronous, shorter interactions, and may break if asked to execute long running jobs.
+
+## Burstiness
+
+Some LLM applications can often be rather bursty. For example, if you put an agent into a spreadsheet-UX, the user could be kicking of thousands of agents at the same time. You need to make sure your infrastructure can handle this burstiness.
+
+This is typically handled by implementing some sort of task queue.
+
+## Double texting
+
+Many times users might interact with your graph in unintended ways. For instance, a user may send one message and before the graph has finished running send a second message.
+
+We call this ["double texting"](double_texting.md) and you will need to think about this when you deploy your agent.
+
+## Checkpointers
+
+Many applications require some level of persistence. An example of this is persistent conversation history, where you can have a conversation one day, and then come back later and resume that conversation.
+
+LangGraph has built in support for this with [checkpointers](persistence.md#checkpoints). When deploying your LangGraph application, you therefor need to also deploy a production-grade checkpointer along side it.
+
+
+## Human-in-the-loop
+
+Long running agents can mess up. A common UX paradigm to overcome this is to add more [human-in-the-loop](human_in_the_loop.md) features to your application.
```

Comment:
* Remove "long running"
* Avoid usage of phrases like "mess up" -- it's colloquial and also requires good knowledge of English to understand. Non native speakers may interpret this incorrectly


```suggestion
Agents can make mistakes. A common UX paradigm to overcome this is to add more [human-in-the-loop](human_in_the_loop.md) features to your application.
```